provider/digitalocean/digital_ocean_test.go

```diff
@@ -143,10 +146,54 @@ func (m *mockDigitalOceanRecordsFail) Records(ctx context.Context, domain string
 	return []godo.DomainRecord{}, nil, fmt.Errorf("Failed to get records")
 }
 
-func TestNewDigitalOceanChanges(t *testing.T) {
-	action := DigitalOceanCreate
-	endpoints := []*endpoint.Endpoint{{DNSName: "new", Targets: endpoint.Targets{"target"}}}
-	_ = newDigitalOceanChanges(action, endpoints)
+// This function is an adapted copy of the testify package's ElementsMatch function with the
+// call to ObjectsAreEqual replaced with cmp.Equal which better handles struct's with pointers to
+// other structs.
+//
+// TODO: Upstream a change to testify package to allow custom matchers.
```

Comment:
```suggestion
```

I think we don't need this TODO in external DNS, but I think this feature would be a good idea 😄

Reply:
Will remove the TODO.